tools/exports.js

```diff
@@ -1,15 +1,4 @@
-exports["Compressor"] = Compressor;
 exports["Dictionary"] = Dictionary;
-exports["JS_Parse_Error"] = JS_Parse_Error;
```

Comment:
> Ah - I see. Is  exception.name  a standard ECMAScript thing?

All the standard `Error` sub-classes has their own `.name`, and the field is also mentioned by MDN:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name